lib/vsc/install/ci.py

```diff
@@ -0,0 +1,163 @@
+#
+# Copyright 2019-2019 Ghent University
+#
+# This file is part of vsc-install,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# the Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/hpcugent/vsc-install
+#
+# vsc-install is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Library General Public License as
+# published by the Free Software Foundation, either version 2 of
+# the License, or (at your option) any later version.
+#
+# vsc-install is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Library General Public License for more details.
+#
+# You should have received a copy of the GNU Library General Public License
+# along with vsc-install. If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Generate configuration files for running CI tests.
+
+Run with: python -m vsc.install.ci
+
+@author: Kenneth Hoste (Ghent University)
+"""
+import logging
+import os
+
+
+JENKINSFILE = 'Jenkinsfile'
+TOX_INI = 'tox.ini'
+
+logging.basicConfig(format="%(message)s", level=logging.INFO)
+
+
+def write_file(path, txt):
+    """Write specified contents to specified path."""
+    try:
+        with open(path, 'w') as handle:
+            handle.write(txt)
+        logging.info("Wrote %s", path)
+    except (IOError, OSError) as err:
+        raise IOError("Failed to write %s in %s: %s" % (path, os.getcwd(), err))
```

Comment:
why is the `in %s` referrring to cwd? if path is relative, prerfix it with cwd before writing